README.md

```diff
@@ -22,6 +22,10 @@ Base URL: http://stats.filspark.com/
 
   http://stats.filspark.com/participants/change-rates
 
+- `GET /participants/scheduled-rewards/?address=<address>&from=<day>&to=<day>`
+
+  http://stats.filspark.com/participants/scheduled-rewards/
```

Comment:
The other URLs in this API docs don't use the trailing slash. It would be nice to be consistent, but it's also not a big deal.